.github/workflows/test-linux.yml

```diff
@@ -48,6 +48,11 @@ jobs:
             PYTHON_VERSION: '2.7'
             TEST_TYPE: 'slow'
     steps:
+      - name: Setup systemd
```

Comment:
@dalthviz, I think you need to add below this line

    shell: bash -l {0}

in order to use a login shell.

@goanpeca?